README.adoc

```diff
@@ -69,21 +82,27 @@ You can modify (add, modify or delete) registry keys, directories and files like
 
 Registry keys:
 
-    ./malboxes.py registry profile modtype key name value valuetype
+    malboxes registry <profile> <modtype> <key name> <value> <valuetype>
 
-    Ex: ./malboxes registry win10_64_analyst add HKCU:\Software Malboxes IsAwesome String
+Example:
+
+    malboxes registry win10_64_analyst add HKCU:\Software "Malboxes IsAwesome" String
```

Comment:
I suck at registry... I thought you were missing a quote.

I'll fix it. Good catch!